config/src/converters/grpc/device.rs

```diff
@@ -4,33 +4,14 @@
 use ::gateway_config::config as gateway_config;
 use gateway_config::TracingConfig as ApiTracingConfig;
 
```

Comment:
THis commit is why we should kill gRPC sooner than later.  Is there a PR to fix gateway-proto?